fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/DppUtils.java

```diff
@@ -90,6 +91,8 @@ public static Class getClassFromColumn(EtlJobConfig.EtlColumn column) throws Spa
                 return Double.class;
             case "DATE":
                 return Date.class;
+            case "BINARY":
+                return BitMap.class;
```

Comment:
What does BitMap.class used for?

Reply:
I wrote it by mistake before, you can delete it.